src/Output.tsx

```diff
@@ -11,7 +11,7 @@ const Output: FunctionComponent<{
   backgroundImageOffsetX: number;
   backgroundImageOffsetY: number;
   text: { text: string; foregroundColour: string; backgroundColour: string }[];
-  overlays: { url: string; width: number; height: number }[];
+  overlays: { url: string; width: number; height: number; scalepos: Array<number> }[];
```

Comment:
I wonder if `[number, number, number, number]` might be a better type here, given it's always explicitly 4 numbers? It might be worth defining a type of `Overlay` tbh given this definition is duplicated below

Reply:
Good points both. I'll look into it.